hack/internal/pkg/remotecommand/remotecommand.go

```diff
@@ -0,0 +1,184 @@
+package remotecommand
```

Comment:
issue: Nothing in this repository makes use of this `remotecommand` package, so it can be removed for now.